lighty-modules/lighty-netconf-sb/pom.xml

```diff
@@ -83,6 +71,19 @@
             <artifactId>singlenode-configuration</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Fix dependencies for correct ones that should have been replaced.

   ietf-netconf -> rfc6241
 https://git.opendaylight.org/gerrit/c/netconf/+/101841/11/model/rfc6241/pom.xml
 
 ietf-netconf-monitoring-> rfc6022
 https://git.opendaylight.org/gerrit/c/netconf/+/101841/11/model/rfc6022/pom.xml#20
 
 ietf-netconf-nmda -> rfc8526
 https://git.opendaylight.org/gerrit/c/netconf/+/101841/11/model/rfc8526/pom.xml#20
 
 netconf-monitoring-extension -> odl-netconf-monitoring 
 https://git.opendaylight.org/gerrit/c/netconf/+/101857

In the commit message, write which dependency was replaced by which one.